sig-instrumentation/README.md

```diff
@@ -81,11 +85,15 @@ The following [subprojects][subproject-definition] are owned by sig-instrumentat
 - **Owners:**
   - https://raw.githubusercontent.com/kubernetes-sigs/metrics-server/master/OWNERS
 ### mutating-trace-admission-controller
+(Deprecated)
```

Comment:
We remove the archived repos from sigs.yaml.

Steering is working on the definition of deprecated and we'll probably introduce a new process for it (@dims has some initial docs).

Reply:
@logicalhan can you remove Heapster and mutating-trace-admission-controller then?

Reply:
done